elastic/apm-server
# Drop transaction.duration.{count,sum.us}

## Motivation/summary

These metrics are not used anywhere, so stop recording them.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Run the server with `-E instrumentation.enabled=true`
2. `curl http://localhost:8200/foo`
3. Wait for breakdown metrics to be indexed
4. Check that there are no documents with `transaction.duration.count` or `transaction.duration.sum.us`

## Related issues

Closes #6176
